examples/xml-attributes/tests/xml_attributes_test.bal

```diff
@@ -11,6 +11,7 @@ int counter = 0;
 }
 public function mockPrint(any... s) {
     if (s[0] is ()) {
+        // Cannot convert () to string.
```

Comment:
Maybe we can eventually improve this test to assert value equality without converting to strings.

Reply:
+1